code/game/objects/items/grenades/grenade.dm

```diff
@@ -86,16 +86,16 @@
 		if(!botch_check(user)) // if they botch the prime, it'll be handled in botch_check
 			arm_grenade(user)
 
-/obj/item/grenade/proc/log_grenade(mob/user, turf/T)
+/obj/item/grenade/proc/log_grenade(mob/user, turf/Turf)
```

Comment:
As I said in your other PR, this does not help.

You need *descriptive* names, not just "turf".

> Unless it is otherwise obvious, try to avoid just extending variables like "C" to "carbon"--this is slightly more helpful, but does not describe the context of the use of the variable.

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
`snake_case` only.